packages/backend/server/src/plugins/copilot/cron.ts

```
@@ -0,0 +1,67 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { Cron, CronExpression } from '@nestjs/schedule';
+
+import { JobQueue, OneDay, OnJob } from '../../base';
+import { Models } from '../../models';
+
+declare global {
+  interface Jobs {
+    'copilot.session.cleanupEmptySessions': {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)